src/Symfony/Component/Notifier/Bridge/Pusher/PusherTransportFactory.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Pusher;
+
+use Pusher\Pusher;
+use Symfony\Component\DependencyInjection\Attribute\Autoconfigure;
+use Symfony\Component\Notifier\Exception\MissingRequiredOptionException;
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Yasmany Cubela Medina <yasmanycm@gmail.com>
+ */
+#[Autoconfigure(tags: ['texter.transport_factory'])]
```

Comment:
Not sure about this @fabpot

Reply:
It won't work, right?

Reply:
it works on our current project. we use texter since it's the most similar of all available. not sure if you want to introduce something new. I'm all ears

Reply:
how should i proceed @OskarStark @fabpot ?

Reply:
Core services are defined explicitly rather than being autoconfigured (to have more control over what the configuration is). You should define the service with the proper tags in FrameworkBundle instead.

Reply:
done @stof